homeassistant/components/tuya/sensor.py

```diff
@@ -734,6 +736,55 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             native_unit_of_measurement=UnitOfElectricPotential.VOLT,
             subkey="voltage",
         ),
+        TuyaSensorEntityDescription(
```

Comment:
Voltage sensors should be disabled by default.

Reply:
Should this have an entity category?

Reply:
How to disable by default ? Sorry but i'm not used with HA and integration, what should I do ?